src/main/pages/che-7/hands-on-guidance/proc_creating-a-workspace-from-code-sample.adoc

```diff
@@ -1,27 +1,131 @@
 [id="creating-a-workspace-from-code-sample_{context}"]
 = Creating a workspace from code sample
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+This section will show how to select a stack from the User Dashboard and start a workspace with its sample code.
+
+It will also explain how to customize a workspace using the User Dashboard.
```

Comment:
if you've started **a** workspace, the later refernce should be to **the** workspace. Thus:

This section walks you through steps to select a stack from the *Dashboard*, start a workspace with a sample project, and customizing that workspace using the *Dashboard* .

Reply:
@supriya-bharadwaj, no walking :) (and Nick is right about the articles).

Reply:
It could be two documents (starting a ws; customizing a ws) but since the most logical workflow for customizing a ws (for a first time user, anyway) is to do so before starting the ws... it makes sense IMHO to have these as a single doc:

* Creating a workspace from code sample

- [ ] Accessing the Dashboard
- [ ] Selecting a stack
- [ ] 
- [ ] 
- [ ] 


-